### ISSUE TITLE
ownCloud client deletes some new local files instead of uploading them

### ISSUE THREAD

**Issue by audreytoskin** [owncloud/client#1644]
## Expected Behavior

After adding new files to the local `owncloud/` folder, the files all should sync to the ownCloud server.
## Actual Behavior

Several months ago I'd synced the photos from a trip to Yosemite National Park. The directory structure was something like `.../photography/Yosemite National Park/June 2013/Day X` with 4 folders for each of the 4 days I was there. Today, I noticed the folders `Day 1` and `Day 2` were completely missing.
## Steps To Reproduce the Error

Here's how I think it started: I haven't used my desktop computer as much lately because I just moved and don't have a proper desk yet. Recently I renamed the parent directory containing `photography`, so when I turned on my desktop computer again and initiated the ownCloud client, just about everything in that renamed folder had to resync. Seems like it would be easier to just rename the synced folder, but I guess ownCloud doesn't know how to do that yet(?). I started this giant resync last night, and since I take a lot of pictures and there thus was a lot of data to resync, I let that run over night.

Today, this afternoon, I noticed the `Day 1` and `Day 2` folders were missing from my Yosemite trip. I logged into the web client, but the folders were missing from the server too. I fortunately had another local copy of the photos, and copied them into the ownCloud photography folders again. But I periodically checked the "activity" tab of the desktop client, and some files were uploaded but some were again deleted for no reason. 

Next, I stopped the sync process, replaced the deleted files one more time, restarted sync in the desktop client, and watched the "activity" tab. Again, some files were uploaded, but others were deleted from the local directory without syncing.
## Server Configuration
- Operating system: Debian Linux, kernel 2.6.32.45-grsec-2.2.2-r3
- Web server: Apache httpd
- Database: MySQL 5.1.56
- PHP version: 5.4.20
- ownCloud server version: 6.0.2
- Storage backend: ...Um, the standard backend?

**installed ownCloud apps:** 

Bookmarks, Calendar, Contacts, Deleted Files, Documents, External Sites, First Run Wizard, Full Text Search, ownCloud Dependencies Info, PDF Viewer, Pictures, Share Files, Tasks, Text Editor, Updater, Versions, Video Viewer, News, Videos

I do not have file encryption enabled, and I do not use a proxy, though I do enforce SSL connections. I am the only user in this ownCloud instance, but I use it to store a lot of data (around 15 GB).
## Client Configuration
- Client version: owncloud-client-1.5.3, installed from from the [openSUSE build service](http://software.opensuse.org/download/package?project=isv:ownCloud:desktop&package=owncloud-client)
- Operating system: Fedora Linux 20, 64-bit, kernel 3.13.4
- OS language: English, US
- Installation path of client: /usr/bin/owncloud
## Log Files

I don't know how long ago the `Day 1` and `Day 2` folders were originally deleted, but I was able to replicated the problem _twice_ today, so I will give you the log files for today. Some of these logs are incredibly long, particularly the desktop client logs, so keep in mind that I first caught the client deleting fles today, 2014-04-06, around around 14:57. I completely shut off the client a little after 16:00 this afternoon, so you're probably mostly interested in the activity within those 2 or 3 hours...

I'm hosting these log files on my personal site because the desktop client logs were so huge, I could not feasibly paste them into a Gist. I literally could not get it to work.
- [ownCloud desktop client log](http://andrew.tosk.in/owncloud-deletion-error-occlient-2014-04-06.log) (warning: file is 106 MB, and hundreds of thousands of lines long! I don't know why it's so big...)
- [ownCloud server log](http://andrew.tosk.in/owncloud-deletion-error-ocserver-2014-04-06.log)
- [Apache https access log for my ownCloud domain](http://andrew.tosk.in/owncloud-deletion-error-apache-https-access.log)
- [Apache https error log for my ownCloud domain](http://andrew.tosk.in/owncloud-deletion-error-apache-https-error.log)
## In Conclusion

I've been [burned](https://github.com/owncloud/mirall/issues/1218) by an ownCloud error causing data loss before, so I'm basically weeping tears of fire and blood right now.
